src/vcf/meta_entry.cpp

```diff
@@ -214,6 +220,22 @@ namespace ebi
             value["Type"] != "String") {
             throw new MetaSectionError{entry.line, "INFO metadata Type is not a Integer, Float, Flag, Character or String"};
         }
+
+        for (const auto & t : info_type) {
+            if (value["ID"] == t.first) {
+                if (value["Type"] != t.second.first) {
```

Comment:
reuse the `number_field` variable (from line 206) for this method, and do the same for `value["Type"]`.